schedule/virtualization/vmware_hyperv.yaml

```diff
@@ -0,0 +1,11 @@
+name:           vmware_hyperv
+description:    >
+    Maintainer: pdostal@suse.cz
+    Testing Kernel on VMware / Hyper-V SLE hosts
+schedule:
```

Comment:
All LGTM. Just curious why do  you prefer this yaml schedule way rather than the old main.pm way?

Reply:
Thank you. I find it more readable and structured. We are slowly adopting it.